images/postgres/main.tf

```
@@ -8,13 +8,22 @@ variable "target_repository" {
   description = "The docker repo into which the image and attestations should be published."
 }
 
+module "dev" { source = "../../tflib/dev-subvariant" }
+
 module "latest" {
   source = "../../tflib/publisher"
 
   target_repository = var.target_repository
   config            = file("${path.module}/configs/latest.apko.yaml")
 }
 
+module "latest-dev" {
+  source = "../../tflib/publisher"
+
+  target_repository = var.target_repository
+  config            = file("${path.module}/configs/latest.apko.yaml")
```

```suggestion
  config            = file("${path.module}/configs/latest.apko.yaml")
  extra_packages = module.dev.extra_packages
```

This injects the dev packages that we lookup in the dev-subvariant module into the published -dev image